deliver/spec/submit_for_review_spec.rb

```diff
@@ -49,28 +49,98 @@ def make_fake_version
     describe :wait_for_build do
       context 'no candidates' do
         let(:fake_app) { make_fake_app }
-        let(:fake_version) { make_fake_version }
         let(:time_now) { Time.now }
+        let(:fake_version) do
+          fake_version = make_fake_version
+          fake_version[:app_version] = "1.2.3"
```

Comment:
Should this maybe moved to `make_fake_version`?

Reply:
Sure, that makes sense. I'll change it right now.

Reply:
Done